indico/modules/events/templates/display/_manage_button.html

```diff
@@ -0,0 +1,132 @@
+           class="notes-editor"
+           data-href="{{ url_for('event_notes.edit', item) }}"
+           data-title="{% trans %}Edit minutes{% endtrans %}">
+            {% if item.has_note -%}
+                {% trans %}Edit minutes{% endtrans %}
+            {%- else -%}
+                {% trans %}Add minutes{% endtrans %}
+            {%- endif %}
+        </a>
+    </li>
+{%- endmacro %}
+
+{% macro _render_dropdown(item, item_type, show_note_operations=false) -%}
+    <ul class="dropdown">
+        {% if item_type == 'EVENT' %}
+            {% if item.can_manage(session.user) %}
+                <li>
+                    <a href="{{ url_for('event_mgmt.conferenceModification', item) }}">
+                        {% trans %}Edit Event{% endtrans %}
+                    </a>
+                </li>
+            {% endif %}
+            {% if item.can_manage_attachments(session.user) %}
+                {% if item.as_legacy.getType() != 'simple_event' %}
+                    {{ _render_note_editor(item) }}
+                    {% if item.scheduled_notes %}
+                        <li>
+                            <a href=""
+                               data-locator="{{ item.locator | tojson | forceescape }}"
+                               class="notes-compile"
+                               data-href="{{ url_for('event_notes.compile', item) }}"
+                               data-title="{% trans %}Edit minutes{% endtrans %}">
+                                {% trans %}Compile Minutes{% endtrans %}
+                            </a>
+                        </li>
+                    {% endif %}
+                {% endif %}
+            {% endif %}
+        {% elif item_type == 'SESSION_BLOCK' %}
+            {% set session_ = item.session %}
+            {% if item.can_manage_attachments(session.user) %}
+                {% if not session_.has_note or show_note_operations %}
+                    {{ _render_note_editor(session_) }}
+                {% endif %}
+                <li>
+                    <a href="" class="js-material-editor"
+                               data-attachment-editor
+                               data-reload-on-change
+                               data-title="{% trans title=session_.title %}Manage materials for '{{ title }}'{% endtrans %}"
+                               data-locator="{{ session_.locator | tojson | forceescape }}">
+                        {%- trans %}Material Editor{% endtrans -%}
+                    </a>
+                </li>
+
+            {% endif %}
+            {% if item.can_manage(session.user) %}
+                <li>
+                    <a href="" data-locator="{{ item.locator | tojson | forceescape }}" class="session-basic-edit">
+                        {% trans %}Edit Session (basic){% endtrans %}
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        {% trans %}Edit Session (full){% endtrans %}
```

Comment:
Hm, I always found it weird that being a session or track manager gave you deep links into management but not a limited view in the management area from where you could also reach the things you have access to.